conans/cli/api/subapi/upload.py

```diff
@@ -62,12 +62,11 @@ def prepare(self, upload_bundle):
         preparator.prepare(upload_bundle)
 
     @api_method
-    def upload_bundle(self, upload_bundle, remote, force=False):
```

Comment:
For me to understand it, why is it removed?

Reply:
Only the argument. It was used only to avoid uploading dirty "scm" packages, but scm doesn't exist anymore.
We will block the "dirty" packages earlier in the process (at export) if anything.

Reply:
Got it! Thanks 😁